cli.js

```diff
@@ -29,6 +29,11 @@ var argv = yargs
   .boolean('commit')
   .default('files', ['README.md'])
   .default('contributorsPerLine', 7)
+  .option('version', {
+    alias: 'v',
+    desc: 'Print installed version',
+    type: 'string'
+  })
   .default('contributors', [])
```

Comment:
This doesn't seem to be the standard way to display version information using yargs:
   https://github.com/yargs/yargs/blob/master/docs/api.md#version

Reading the docs, I think that this should be enough:
```
  .alias('v', 'version')
  .version()
```

Also, could you move this block right under `alias('h', 'help')`? It would make more sense ;)